kafka-client/build.gradle.kts

```diff
@@ -62,6 +62,19 @@ tasks.publish {
     dependsOn(tasks[writeVersionPropertiesFile])
 }
 
+configurations {
```

Comment:
I'll admit I had to ask chatGPT how to access the stuff in the testutils of `kafka-client` from the tests in `responsive-test-utils`, and this was how it said to do it. Not sure it's necessary the right way to do so, if you have a different suggestion

Reply:
🤷 when it comes to build systems, if it works it works